db/create-sandbox-schema.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="create-sandbox" author="frolov">
+        <sql>
+            CREATE SCHEMA IF NOT EXISTS save_sandbox;
```

Comment:
database is schema for mysql: https://stackoverflow.com/questions/11618277/difference-between-schema-database-in-mysql

we don't need to create schema separately, because now it's created by url

Reply:
Yes. Removed it